<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryCardEditorDialog</class>
 <widget class="QWidget" name="MemoryCardEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Card Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/duckstation-qt.qrc">
    <normaloff>:/icons/duck.png</normaloff>:/icons/duck.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="cardA">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Icon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>File Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Blocks</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Memory Card 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cardBPath">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="newCardB">
       <property name="text">
        <string>New...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/duckstation-qt.qrc">
         <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openCardB">
       <property name="text">
        <string>Open...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/duckstation-qt.qrc">
         <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <widget class="QLabel" name="cardBUsage">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBoxB">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::NoButton</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
     <item>
      <widget class="QLabel" name="cardAUsage">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBoxA">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::NoButton</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Memory Card 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cardAPath"/>
     </item>
     <item>
      <widget class="QToolButton" name="newCardA">
       <property name="text">
        <string>New...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/duckstation-qt.qrc">
         <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openCardA">
       <property name="text">
        <string>Open...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/duckstation-qt.qrc">
         <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <widget class="QTableWidget" name="cardB">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Icon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>File Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Blocks</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="centerButtonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/duckstation-qt.qrc"/>
 </resources>
 <connections/>
</ui>
